data_steward/cdr_cleaner/cleaning_rules/clean_height_weight.py

```diff
@@ -217,48 +248,48 @@
     measurement_id,
     adj_Ht,
     adj_unit
-  FROM `{project_id}.{sandbox_dataset_id}.{height_table}`
+  FROM `{{project_id}}.{{sandbox_dataset_id}}.{{height_table}}`
   WHERE f_outlier_Ht=0
     AND f_disagree2_Ht=0
     AND adj_Ht IS NOT NULL
     AND adj_unit=8582
 )
-JOIN `{project_id}.{dataset_id}.measurement` m USING (measurement_id)
-LEFT JOIN `{project_id}.{dataset_id}.concept` u_c ON (adj_unit=concept_id)
-"""
+JOIN `{{project_id}}.{{dataset_id}}.measurement` m USING (measurement_id)
+LEFT JOIN `{{project_id}}.{{dataset_id}}.concept` u_c ON (adj_unit=concept_id)
+""")
 
-DELETE_HEIGHT_ROWS_QUERY = """
-DELETE
-FROM `{project_id}.{dataset_id}.measurement` m
-WHERE measurement_id IN
-(SELECT measurement_id
-FROM `{project_id}.{dataset_id}.measurement` m
-LEFT JOIN `{project_id}.{dataset_id}.measurement_ext` me
-USING (measurement_id)
-WHERE (m.measurement_concept_id IN (3036277, 3023540, 3019171))
-AND me.src_id != 'PPI/PM')
-"""
+DELETE_HEIGHT_ROWS_QUERY = jinja_env.from_string("""
+    SELECT * 
+    FROM `{{project_id}}.{{dataset_id}}.measurement` AS m
+    WHERE measurement_id IN
+    (SELECT measurement_id 
+    FROM `{{project_id}}.{{dataset_id}}.measurement` AS m
+    LEFT JOIN `{{project_id}}.{{dataset_id}}.measurement_ext` AS me
+    USING (measurement_id)
+    WHERE (m.measurement_concept_id IN (3036277, 3023540, 3019171))
+    AND me.src_id != 'PPI/PM')
```

Comment:
I think this might be the issue: The select statement should select everything except the rows to be deleted, but here it is only selecting the rows to be deleted.

Reply:
I changed it to this:

```python
DELETE_WEIGHT_ROWS_QUERY = jinja_env.from_string("""
    SELECT * FROM `{{project_id}}.{{dataset_id}}.measurement`
    WHERE measurement_id NOT IN
    (SELECT measurement_id
    FROM `{{project_id}}.{{dataset_id}}.measurement` AS m
    LEFT JOIN `{{project_id}}.{{dataset_id}}.measurement_ext` AS me
    USING (measurement_id)
    WHERE (m.measurement_concept_id IN (3025315, 3013762, 3023166))
    AND me.src_id != 'PPI/PM')
"""```

But getting strange results. After I run `clean_height_weight.py` and query `measurement` the resulting query still has `measurement_concept_id` of 3025315 and 3036277. Not sure if this is because of another query that is ran. Also, sandboxed tables aren't getting populated still. Looking more into why that is happening.

Reply:
You shouldn't have to change the concept_ids, the negation should suffice.

Reply:
Okay so there must be another reason why `measurement_concept_ids` 302513 and 3036277 aren't dropped.

Reply:
@dcampbell-vumc I think this is still an issue, which is preventing to get the data for height_table.